src/modules/land_detector/LandDetector.h

```diff
@@ -154,26 +153,33 @@ class LandDetector : public ModuleBase<LandDetector>, px4::ScheduledWorkItem
 	systemlib::Hysteresis _ground_contact_hysteresis{true};
 	systemlib::Hysteresis _ground_effect_hysteresis{false};
 
-	struct actuator_armed_s	_arming {};
+	actuator_armed_s _arming {};
 
 private:
-	void Run() override;
 
-	void _check_params(bool force = false);
+	void _check_params();
+
+	void Run() override;
 
 	void _update_state();
 
-	param_t _p_total_flight_time_high{PARAM_INVALID};
-	param_t _p_total_flight_time_low{PARAM_INVALID};
+	bool _previous_armed_state{false}; ///< stores the previous actuator_armed.armed state
+
+	uint32_t _measure_interval{LAND_DETECTOR_UPDATE_INTERVAL};
```

Comment:
This is not used.

Reply:
Removed!  Thanks for spotting that as well!  Rebased on current master and added a commit that fixes the issues you identified. Thanks again!